Project.toml

```diff
@@ -0,0 +1,16 @@
+name = "CLFFT"
+uuid = "fe1fc2a7-b730-4eae-aba2-587af0173dc9"
+authors = []
+version = "0.1.0"
```

Comment:
Please make sure to use the correct version for the package, based on the latest version that was registered. This is a breaking update so you need to bump the minor version.